extensions/united-nations/package.json

```
@@ -213,4 +213,4 @@
     "prettier": "^3.5.2",
     "typescript": "^5.7.3"
   }
-}
+}
```

```suggestion
}

```

Please bring the final newline back.